sdk/core/core-amqp/src/retry.ts

```diff
@@ -59,6 +63,12 @@ export interface RetryConfig<T> {
    * Extremely useful in providing better debug logs.
    */
   operationType: RetryOperationType;
+  /**
+   * @property {number} operationTimeoutInMs Maximum time allowed for within which an operation is
+   * expected to be completed. If operation takes longer, it will be cancelled, considered as failed
+   * with a retryable error and will be retried again on based on supplied `RetryConfig`
+   */
+  operationTimeoutInMs?: number;
```

Comment:
@bterlson I thought you had earlier decided not to add the `Ms` suffix as milliseconds is to be the default?

Reply:
I had said that. However:

* `ms` suffix exists in a few places still
* Ramya and others have argued the suffix is useful

I found myself agreeing with the latter - I can see the lack of suffix causing issues in a usability lab, especially for people who aren't super familiar with JS and its canonical usage of Ms everywhere. So it's the safer assumption to go with, and we can change if we hear feedback. I've updated [the guidelines](https://azuresdkspecs.z5.web.core.windows.net/TypeScriptSpec.html#ts-options) appropriately.